### DISCUSSION TITLE
Arrows on Keyboard?

### DISCUSSION THREAD

**Discussion by cneville23** [florisboard/florisboard#2576] [Feedback]
Hi, I've used Florisboard for a few years now because I like having arrows on the on-screen keyboard. Is there a way to have these always shown on the first screen, not hidden in the overflow menu? It's not clear to me if it is possible to get back having several rows of function buttons in the settings menu.

Thanks!

**Comment by BHydden** [comment]
Tap the `...` menu in the top right of the keyboard, then scroll to the bottom of the list and click `Reorder actions`

**Reply by cneville23** [reply]
Thanks!
